cub/test/catch2_test_device_transform.cu

```diff
@@ -388,6 +389,12 @@ C2H_TEST("DeviceTransform::Transform add five streams", "[device][device_transfo
   c2h::device_vector<std::int64_t> d(num_items, 4);
   c2h::device_vector<float> e(num_items, 5);
 
+  c2h::gen(C2H_SEED(1), a, std::int8_t{10}, std::int8_t{100});
```

Comment:
suggestion: you are likely ending up with a single tile of input. Also randomizing `num_items` in a bit wider range would serve the goal of this PR.

Reply:
Oh yes, I made a similar change in #4815 but forgot to extract it to this PR as well. I went with a generator expression: `const int num_items = GENERATE(100, 100'000);`. Let me know if you prefer a random size instead.